sql-plugin/src/main/scala/org/apache/spark/rapids/hybrid/HybridExecutionUtils.scala

```diff
@@ -49,17 +55,26 @@ object HybridExecutionUtils extends PredicateHelper {
    * Determine if the given FileSourceScanExec is supported by HybridScan.
    */
   def useHybridScan(conf: RapidsConf, fsse: FileSourceScanExec): Boolean = {
+    // Hybrid scan can be enabled either by setting the global SQL config(HYBRID_PARQUET_READER)
+    // or by adding the specific SQL hint(HYBRID_SCAN_HINT) upon desired tables.
     if (!conf.useHybridParquetReader) {
-      return false
+      if (fsse.relation.options.contains(HybridExecOverrides.HYBRID_SCAN_TAG)) {
+        logInfo(fsse.nodeName + " carries SQL HINT(" + HybridExecOverrides.HYBRID_SCAN_HINT +
```

Comment:
logWarning better since it could be an unexpected behavior?

Reply:
It might be a little bit confusing here. This condition stands for the global hybridScan config is disabled while  HYBRID_SCAN_HINT is detected, which is expected.